### ISSUE TITLE
task050_multirc_answerability.json skew 

### ISSUE THREAD

**Issue by danyaljj** [allenai/natural-instructions#619]
Has a massive skew: 
```
   📋 classes: ['No.' 'Yes.'] 
   📋 Norm_counts: [0.94815385 0.05184615] 
   📋 Distribution of classes: [6163  337] 
   📊 entropy= 0.29418811830775965
```
@swarooprm happen to have the script for collecting the instances of this dataset?

**Comment by Palipoor** [comment]
I will look into this.

**Comment by Palipoor** [comment]
There's a problem that I can't figure out. It seems to me that the answerability data is extracted from multirc by using sentences from the passages paired with questions. But, when I was trying to make sure this is how the data is gathered, I found out that there's almost no overlap between the questions in this file and the multirc data! I checked both the data on their own website and the data from superglue. I don't know what I'm doing wrong. I can go on and create an answerability task from the MultiRC but then it won't have the same instances as the task050 file.
